### PR TITLE
:lipstick: feat: Sticker block redesign block redesign #638

### LINKED ISSUE #638: :lipstick: Sticker Block Redesign: block redesign
## User stories

As a user, I want the "Sticker Block" component in the app to be redesigned to enhance its usability and visual appeal.

---

Quick links: [Summary](#summary) | [Description](#description) | [Template](#template) | [Example](#example) | [Resources](#resources)

---

### Technical Description
The current design is: 
image

It should be changed to:
image

The files can be found at `libs/features/convs-mgr/stories/blocks/library/sticker-message-block/`. 

The designs are [here](https://www.figma.com/file/ZGHzSstXOYPMFU1EiqjEHD/Untitled?type=design&node-id=1%3A4&mode=design&t=Hf7DxaUUMQ3BWy57-1)

### Acceptance Criteria

1. The block should look exactly like the designs.
2. Use of `rem`, `em` as opposed to `px`.
3. All changes should be isolated to the following  `libs/features/convs-mgr/stories/blocks/library/sticker-message-block/`.

### PR DESCRIPTION
**Pull request by thadeusdev**
# Description

- changed the stick-note icon and using its svg equivalent.
- Styled the font-weight and font-size for the block-title.
- changes ellipsis-v expand menu icons and used its equivalent svg.
- Changed the Plus icon to stick note svg and styled it including it background color and border radius(in rem).
- Styled/redesigned sticker link input field.
- redesigned and positioned jsPlumb

Fixes #638

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Screenshot (optional)
Screenshot from 2023-10-24 16-47-57

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] My changes generate no new warning dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: libs/features/convs-mgr/stories/blocks/library/sticker-message-block/src/lib/components/sticker-block/sticker-block.component.html [hunk 1]
```diff
@@ -1,29 +1,63 @@
...
+          class="fileSrc"
+          accept=".svg, .gif, .webp"
+          (change)="processSticker($event)"
+        />
+      </div>
+    </div>
 
-		<div [id]="stickerInputId" class="sticker_bg" [ngClass]="byPassedLimits.length ? 'error' : ''">
-			<div (click)="stickerInput.click()" class="sticker-icon" *ngIf="!this.block.fileSrc && !isLoadingSticker">
-				<i class="fas fa-plus"></i>
```
Comment (LemmyMwaura) [inline libs/features/convs-mgr/stories/blocks/library/sticker-message-block/src/lib/components/sticker-block/sticker-block.component.html:6]: Turn off your indents when saving @thadeusdev

**Review by LemmyMwaura — CHANGES_REQUESTED** [review]
Sync the latest changes and then resolve conflicts. Also, kindly edit your PR scope on your title as this is a design feature and not ci related.

File: libs/features/convs-mgr/stories/blocks/library/sticker-message-block/src/lib/components/sticker-block/sticker-block.component.html (hunk 1 above)
Comment (thadeusdev) [reply libs/features/convs-mgr/stories/blocks/library/sticker-message-block/src/lib/components/sticker-block/sticker-block.component.html:6]: noted. Thank you for recommendation 🙏

**Comment by LemmyMwaura** [comment]
Closing this as its been resolved

**Pull request closed without merging**